### DISCUSSION TITLE
Install from Marketplace

### DISCUSSION THREAD

**Discussion by MichaelCurrin** [MichaelCurrin/auto-commit-msg#60] [General]
The extension is now available via the Install button here.

https://marketplace.visualstudio.com/items?itemName=MichaelCurrin.auto-commit-msg

That is [v0.22.0](https://github.com/MichaelCurrin/auto-commit-msg/releases/tag/v0.22.0).

Clean-up is still needed to the docs and Wiki before this can get a major release of v1.0.0 - any requests on doc layout or content would be appreciated.

**Comment by MichaelCurrin** [comment]
@glassesUSA please try this out.

**Comment by MichaelCurrin** [comment]
@JoanNgatia FYI

**Comment by glassesUSA** [comment]
Auto updated automatically for me without having to do anything :o
